packages/p2p/src/stores/order-store.js

```diff
@@ -280,6 +392,39 @@ export default class OrderStore {
         }
     }
 
+    @action.bound
+    verifyEmailVerificationCode(verification_action, verification_code) {
+        if (verification_action === 'p2p_order_confirm') {
+            requestWS({
+                p2p_order_confirm: 1,
+                id: this.order_id,
+                verification_code,
+                dry_run: 1,
+            }).then(response => {
+                this.setIsLoadingModalOpen(false);
+                if (response) {
+                    if (!response.error) {
+                        clearTimeout(wait);
+                        const wait = setTimeout(() => this.setIsEmailLinkVerifiedModalOpen(true), 600);
+                    } else if (
```

Comment:
```suggestion
                    } else if (['InvalidVerificationToken', 'ExcessiveVerificationRequests'].includes(response.error.code))
```

Reply:
As for the Array.includes, it's a smaller statement but my (future) plan is to create a const file with all error codes, that would be cleaner.